main.py

```diff
@@ -0,0 +1,68 @@
+import tensorflow as tf
+import pickle as pkl
+import config
+import numpy as np
+from utils import draw_subspectrogram
+
+def main() :
+
+    # import data
+    with open(config.EMOTIFY_SPECTROGRAM_DUMP_PATH,"rb") as f:
```

Comment:
For information, I am adding a `load_dump` function in my PR #3 that could be used here, and below.